std_msgs/Header header
wireless_network/WirelessNetwork[] networks